addons/dialog/functions/fnc_gui_slider.sqf

```diff
@@ -22,11 +22,36 @@
  */
 
 params ["_controlsGroup", "_defaultValue", "_settings"];
-_settings params ["_min", "_max", "_formatting", "_isPercentage"];
+_settings params ["_min", "_max", "_formatting", "_isPercentage", "_drawRadius", "_radiusCenter", "_radiusColor"];
 
 private _ctrlSlider = _controlsGroup controlsGroupCtrl IDC_ROW_SLIDER;
 private _ctrlEdit = _controlsGroup controlsGroupCtrl IDC_ROW_EDIT;
 
+if (_drawRadius) then {
+    [missionNamespace, "Draw3D", {
+        _thisArgs params ["_ctrlSlider", "_center", "_color"];
+
+        if (isNull _ctrlSlider || {_center isEqualTo objNull}) exitWith {
+            removeMissionEventHandler [_thisType, _thisID];
+        };
+
+        if (_center isEqualType objNull) then {
+            _center = ASLToAGL getPosASLVisual _center;
+        };
+
+        private _radius = sliderPosition _ctrlSlider;
+        private _count = CIRCLE_DOTS_MIN max floor (2 * pi * sqrt _radius / CIRCLE_DOTS_SPACING);
```

Comment:
The dots are a bit sparse now. Consider doing `_radius ^ 0.65` instead of `sqrt` (or maybe another value between 0.6 and 0.7).